iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/match/AffMatchResultChooser.java

```diff
@@ -13,7 +13,7 @@
  * @author Łukasz Dumiszewski
 */
 
-class AffMatchResultChooser implements Serializable {
+public class AffMatchResultChooser implements Serializable {
```

Comment:
This is now always called with equal organization ids. Should be simplified, or not?